intersphinx_registry/registry.json

```
@@ -11,10 +11,10 @@
   "bokeh": ["https://docs.bokeh.org/en/latest/", null],
   "boltons": ["https://boltons.readthedocs.io/en/latest/", null],
   "bottle": ["https://bottlepy.org/docs/dev/", null],
-  "build": ["https://pypa-build.readthedocs.io/en/latest/", null],
+  "build": ["https://build.pypa.io/en/latest/", null],
   "cartopy": ["https://scitools.org.uk/cartopy/docs/latest/", null],
   "cffi": ["https://cffi.readthedocs.io/en/latest/", null],
-  "click": ["https://click.palletsprojects.com/", null],
+  "click": ["https://click.palletsprojects.com/en/8.1.x/", null],
```

I think some of those permanently moved are wrong, we def don't want to point to a given version but the latest stable  at doc build time. ie if click `8.2.x` is published, I don't think we would want intersphinx_registry to have to be updated.

```suggestion
  "click": ["https://click.palletsprojects.com/", null],
```